### PR TITLE
Wix case study

### LINKED ISSUE #4598: content: How Wix Improved Websites Performance by Evolving Their Infrastructure [2021-02-05]
[draft](https://docs.google.com/document/d/10rNsEFGNopmysKJKivfZsVwZzCeh7rw0fb5qK9htSR8/edit#)

#### Provide a brief title for your content proposal

How Wix Improved Websites Performance by Evolving Their Infrastructure

#### Provide a brief summary of your content proposal

This is the tale of some major changes we introduced at Wix over the years to improve site loading performance through leveraging industry standards, cloud providers, and CDN capabilities.
Performance remains a very important focus for us across the company, and many improvements are still being worked on and are gradually being rolled out to users.

Many of these solutions and best standards can and should be applied to any website in general.

#### What content already exists on this topic?

None, but some of the topics are discussed in content such as https://web.dev/content-delivery-networks/

#### Who will be authoring the content?

https://github.com/alonkochba

#### Are there any stakeholders who should be informed when this content changes?

https://github.com/alonkochba

#### What area is your content related to?

Performance

#### What kind of content are you creating?

Case Study / Blog Post

### PR DESCRIPTION
**Pull request by kaycebasques**
Fixes #4598

@alonkochba I'll let you know when this is ready.

### CODE REVIEW

**Comment by kaycebasques** [comment]
@alonkochba please add a comment with `@googlebot I consent` so that the CLA bot stops barking at us

File: src/site/content/en/blog/wix/index.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
+---
+layout: post
+title: How Wix improved website performance by evolving their infrastructure
+subhead: >
+  A case study of some major changes introduced at Wix to improve website loading
+  performance for millions of sites, clearing the path for them to receive good PSI and
```
Comment (kaycebasques) [inline src/site/content/en/blog/wix/index.md:6]:
```suggestion
  performance for millions of sites, clearing the path for them to receive good PageSpeed Insights and
```

**Comment by alonkochba** [comment]
@googlebot I consent

@kaycebasques thanks, going over the small changes you did now

**Comment by alonkochba** [comment]
@googlebot I consent.

File: src/site/content/en/blog/wix/index.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+tags:
+  - blog
+  - fast
+  - performance
+  - web-vitals
+  - case-study
+---
+
+{% Aside %}
+  Alon leads the backend team at Wix.
```
Comment (alonkochba) [inline src/site/content/en/blog/wix/index.md:26]:
```suggestion
  Alon leads the core backend team at [Wix](https://www.wix.com).
```

File: src/site/content/en/blog/wix/index.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+</figure>
+
+DebugBear recently released a very interesting [Website Builder Performance
+Review](https://www.debugbear.com/blog/website-builder-performance-review),
+which touches on some of the areas I mentioned above and examines the
+performance of very simple sites built on each platform. This
+[site](https://matt05041.wixsite.com/bizsolutions) was built almost two years
+ago, and not modified since, but the platform is continually improving, and the
+site performance along with it, which can be witnessed by [viewing its
+data](https://www.debugbear.com/project/175/pageLoad/873/overview?dateRange=2019-03-31T21%3A00Z-to-2021-02-10T21%3A59Z)
```
Comment (alonkochba) [inline src/site/content/en/blog/wix/index.md:371]:
```suggestion
data](https://www.debugbear.com/project/175/pageLoad/873/overview?dateRange=2019-03-31T21%3A00Z-to-2021-03-31T21%3A59Z)
```

**Review by alonkochba — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/wix/index.md [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+compression improvements with almost no trade-offs, and is slowly becoming more
+popular, as described in the yearly Web Almanac [Compression
+chapter](https://almanac.httparchive.org/en/2020/compression#what-type-of-content-should-we-compress).
+It has been supported by [all the major browsers](https://caniuse.com/brotli)
+for a while.
+
+We enabled Brotli support on our nginx proxies in the edges, for all clients
+that support it.
+
+Moving to use Brotli compression reduced our median file transfer sizes by **21%
```
Comment (alonkochba) [inline src/site/content/en/blog/wix/index.md:294]: Can't break line here - it breaks the bold of 21%-25% (see in deploy preview)

Comment (kaycebasques) [reply src/site/content/en/blog/wix/index.md:294]: Sharp eye, thanks for catching

File: src/site/content/en/blog/wix/index.md [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+- 25%** resulting in a reduced bandwidth usage and improved loading times.
```
Comment (kaycebasques) [inline src/site/content/en/blog/wix/index.md:295]:
```suggestion
Moving to use Brotli compression reduced our median file transfer sizes by **21% to 25%**
resulting in a reduced bandwidth usage and improved loading times.
```

**Pull request merged**